docs/basics/ExampleTodoList.md

```diff
@@ -9,116 +9,85 @@ This is the complete source code of the tiny todo app we built during the [basic
 ```js
 import React from 'react'
 import { render } from 'react-dom'
-import { createStore } from 'redux'
 import { Provider } from 'react-redux'
-import App from './containers/App'
+import { createStore } from 'redux'
 import todoApp from './reducers'
+import App from './components/App'
 
 let store = createStore(todoApp)
```

Comment:
I'd rather change it below. It should be just `store` to avoid implying that we have many stores.